src/labels/CustomLabels.labels

```diff
@@ -1540,7 +1540,7 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>RD2_EnablementMigrationWarning</shortDescription>
-        <value>WARNING: This process will rearrange existing data, insert new records, and delete records that don't match the Enhanced Recurring Donations model. This process may take some time. You can leave this page during migration and return later to check the progress.</value>
+        <value>&lt;b&gt;WARNING: This process will rearrange existing data, insert new records, and delete records that don't match the Enhanced Recurring Donations model. This process may take some time. You can leave this page during migration and return later to check the progress.&lt;/b&gt;</value>
```

Comment:
@sam-knox I'm not sure of this part of the message: `and delete records that don't match the Enhanced Recurring Donations model`. I don't believe we're ever planning on deleting Recurring Donation records, though we _may_ delete future installment opportunities beyond the _next_ one. The way I read this message is that it will delete my Recurring Donation records if they dont match our RD2 model (though what exactly would define not meeting the RD2 model is unclear).

Reply:
@force2b - Yeah, good call. That message was written when we were talking about deleting future open Opps during migration. I never went back and updated the message when we decided not to do that. For that matter we aren't inserting and new records either (that I know of).

How about this as a re-written message:

"WARNING: This process rearranges existing Recurring Donation data to match the Enhanced Recurring Donations model. This process may take some time. You can leave this page and return later to check the progress."

Reply:
I like that @sam-knox. Thanks!

Reply:
@sam-knox , the data migration warning message change has been done. 
@force2b , the PR is ready for review.